Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "MUXSDKStats",
+    platforms: [
+        .iOS(.v9),
+        .tvOS(.v9)
+    ],
+    products: [
+        .library(name: "MUXSDKStats", targets: ["MUXSDKStatsTargets"])
+    ],
+    dependencies: [
+        .package(
+            name: "MuxCore",
+            url: "https://github.com/AndrewBarba/stats-sdk-objc.git",
+            .branch("master")
+        )
+    ],
+    targets: [
+        .binaryTarget(
```

Comment:
SwiftPM doesn't support binary targets with dependencies so we get around this with a fancy trick I stole from here: https://forums.swift.org/t/swiftpm-binary-target-with-sub-dependencies/40197